assets/js/search.js

```diff
@@ -30,17 +29,28 @@ setDateToday();
     )
 })(jQuery);
 
-$(document).ready(function CheckPreviousLocation() {
-  const elem = document.getElementById("location");
-  if (sessionStorage.getItem("locationPerm")) {
-    console.log(
-      `Set the Location based on PageLoad...` +
-        sessionStorage.getItem("locationPerm")
-    );
-    elem.value = sessionStorage.getItem("locationPerm");
-  }
+document.getElementById("logout-confirm-btn").addEventListener(
+    "click", logOut
+)
+
+const FORM = document.getElementById("main-form");
+const STORAGE_ITEM = "location";
+const LOCATION_INPUT = document.getElementById("location");
```

Comment:
```suggestion
const LOCATION_INPUT = document.getElementById(STORAGE_ITEM);
```

Reply:
Will not adopt because the location happens to be the same name of the ID of the location input element.